cg/cli/clean.py

```diff
@@ -137,6 +137,46 @@ def scoutauto(context, days_old: int, yes: bool = False, dry_run: bool = False):
         context.invoke(scout, bundle=bundle, yes=yes, dry_run=dry_run)
 
 
+@clean.command("prune-hk")
+@click.option("-c", "--case-id", type=str)
+@click.option("-t", "--tags", multiple=True)
+@click.option("-y", "--yes", is_flag=True, help="skip checks")
+@click.option(
+    "-d", "--dry-run", is_flag=True, help="Shows cases and files that would be cleaned"
+)
+@click.pass_context
+def prune_hk(context, case_id, tags, yes, dry_run):
+    """ Remove files found in older housekeeper bundles """
+    if case_id:
+        cases = [context.obj["db"].family(case_id)]
+    else:
+        cases = context.obj["db"].families()
+    for case in cases:
+        case_id = case.internal_id
+        last_version = context.obj["hk"].last_version(bundle=case_id)
+        LOG.info("Searching %s bundle for outdated files", case_id)
+        hk_files = []
+        for tag in tags:
+            hk_files.extend(context.obj["hk"].get_files(bundle=case_id, tags=[tag]))
+        for hk_file in hk_files:
+            file_path = Path(hk_file.full_path)
+            if hk_file.version_id == last_version.id:
+                continue
+            if os.stat(file_path).st_nlink > 1:
```

Comment:
Why not? We will leave some files elsewhere, but is that not ok? They should be removed from our production flow, by this process or other processes?

Reply:
You are probably right! I added this assuming that maybe we have no good overview of where the files are linked.